icarus_backend/clearance/ClearanceViews.py

```diff
@@ -0,0 +1,103 @@
+from .ClearanceModel import Clearance
+from django.utils import timezone
+from django.http import HttpResponse
+from django.core.serializers.json import DjangoJSONEncoder
+import json
+import uuid
+
+def add_clearance(request):
+    request = json.loads(request.body)
+    clearance_id = uuid.uuid4()
+    created_by = request['created_by']
+    message = request['message']
+    state = request['state']
```

Comment:
Maybe check and make sure it's the proper state? Also maybe try to figure out how to limit the allowed inputs of the clearance state to 'PENDING', 'ACCEPTED', 'DENIED'?